### PR TITLE
doc: add minutes for meeting 12 Jan 2017

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2017-01-12/minutes.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+Unfortunately youtube live did not co-operate so we don't have a recording.
+
+## Agenda
+* Stand up
+* Actions from last meeting
+* Review https://github.com/nodejs/benchmarking/pull/55 and talk about how to fill in gaps
+* Review https://github.com/nodejs/benchmarking/issues/86
+* Review https://github.com/nodejs/benchmarking/issues/63
+* https://github.com/nodejs/node/issues/1409
+* discus about Node new Node workload
```
Comment (kunalspathak) [inline meetings/2017-01-12/minutes.md]: nit: seems like there are typos in this sentence?

**Comment by kunalspathak** [comment]
LGTM

**Comment by mhdawson** [comment]
Addressed comment

**Comment by mhdawson** [comment]
@gareth-ellis can I get a review so that I can land ?

**Comment by mhdawson** [comment]
Landed as 306d4c9b6959094fb1576fd175ab6d28bac499aa

**Pull request closed without merging**